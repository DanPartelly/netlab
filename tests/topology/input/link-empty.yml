defaults:
  device: iosv
  inventory: dump

nodes:
  e1:
  e2:

links:
