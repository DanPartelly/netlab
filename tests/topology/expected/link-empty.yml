input:
- topology/input/link-empty.yml
- package:topology-defaults.yml
links: []
name: input
nodes:
  e1:
    box: cisco/iosv
    device: iosv
    id: 1
    loopback:
      ipv4: 10.0.0.1/32
    mgmt:
      ifname: GigabitEthernet0/0
      ipv4: 192.168.121.101
      mac: 08-4F-A9-00-00-01
    name: e1
  e2:
    box: cisco/iosv
    device: iosv
    id: 2
    loopback:
      ipv4: 10.0.0.2/32
    mgmt:
      ifname: GigabitEthernet0/0
      ipv4: 192.168.121.102
      mac: 08-4F-A9-00-00-02
    name: e2
provider: libvirt
